ci_scripts/ci_post_clone_sub_scripts/app_settings.sh

```
@@ -0,0 +1,68 @@
+
+# 스크립트 실행 위치를 고정
+SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
+cd "$SCRIPT_DIR/../.."
+
+# xcconfigs 디렉토리 생성 (존재하지 않을 경우)
+mkdir -p ./Projects/App/xcconfigs
+chmod -R 777 ./Projects/App/xcconfigs
+
+# 필수 환경 변수 확인
+check_env_var() {
+    if [ -z "${!1}" ]; then
```

Shellcheck (0.10.0)

[warning] In POSIX sh, indirect expansion is undefined.

(SC3053)